src/core/thread/mle_router.cpp

```diff
@@ -988,6 +989,7 @@ otError MleRouter::HandleLinkAccept(const Message &         aMessage,
     router->SetKeySequence(aKeySequence);
 
     Signal(OT_NEIGHBOR_TABLE_EVENT_ROUTER_ADDED, *router);
+    OtnsStatus("router_added=%s", router->GetExtAddress().ToString().AsCString());
```

Comment:
Can we use either `OT_NEIGHBOR_TABLE_EVENT_ROUTER_ADDED`? or `otThreadRegisterNeighborTableCallback()`